### PR TITLE
Add Tooltip Component

### PR DESCRIPTION
**Pull request by kwood15**
This is WIP at the min so any feedback is welcome cheers

screen shot 2017-09-22 at 18 04 27

### CODE REVIEW

File: nebula-components/components/_tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    z-index: 1;
+    max-width: calc(100vw - #{$nb-spacing-unit-double});
+    transform: scale(0);
+    color: $nb-color-white;
+    background-color: $nb-color-alpha;
+    padding: $nb-spacing-unit;
+  }
+
+  .#{$nb-namespace}c-tooltip__toggle {
+    background-color: transparent;
```
Comment (rbrtsmith) [inline nebula-components/components/_tooltip.scss]: Don't need this.

File: nebula-components/components/_tooltip.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    transform: scale(1);
+  }
+
+  .#{$nb-namespace}c-tooltip__content {
+    position: absolute;
+    z-index: 1;
+    max-width: calc(100vw - #{$nb-spacing-unit-double});
+    transform: scale(0);
+    color: $nb-color-white;
+    background-color: $nb-color-alpha;
```
Comment (rbrtsmith) [inline nebula-components/components/_tooltip.scss]: `$nb-color-black`

File: demo/src/Components/Tooltip/Tooltip.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+const Tooltip = ({ isOpen, id, handleClick }) => (
+  <div className="o-grid__item u-1/4@sm">
+    <div className={classNames('c-tooltip', { 'is-open': isOpen })}>
+      <button className="c-tooltip__toggle" onClick={() => { handleClick(id) }}>toggle me: {id}</button>
```
Comment (rbrtsmith) [inline demo/src/Components/Tooltip/Tooltip.jsx]: don't need any kind of toggle button as we'll be using `:hover` and `:focus` for tooltips.
Which also means we don't need any state etc.

File: demo/src/Components/Tooltip/index.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      },
+    }
+  }
+
+  handleClick = (direction) => {
+    const newState = Object.keys(this.state.openTooltip).reduce((acc, curr) => {
+      acc[curr] = false
+      return acc
+    }, {})
+    this.setState({
```
Comment (rbrtsmith) [inline demo/src/Components/Tooltip/index.jsx]: For the demo just put two directions as static props on two examples.  No need to mess around with state or anything here.

**Review by rbrtsmith — CHANGES_REQUESTED** [review]
See above comments

File: demo/src/Components/Tooltip/Tooltip.jsx (hunk 3 above)
Comment (kwood15) [reply demo/src/Components/Tooltip/Tooltip.jsx]: done

File: demo/src/Components/Tooltip/index.jsx (hunk 4 above)
Comment (kwood15) [reply demo/src/Components/Tooltip/index.jsx]: done

File: nebula-components/components/_tooltip.scss (hunk 1 above)
Comment (kwood15) [reply nebula-components/components/_tooltip.scss]: done

File: nebula-components/components/_tooltip.scss (hunk 2 above)
Comment (kwood15) [reply nebula-components/components/_tooltip.scss]: done

**Comment by kwood15** [comment]
Have refactored and here are some screenshots based - tested with changing the grid:

screen shot 2017-09-25 at 13 03 45
screen shot 2017-09-25 at 13 04 55
screen shot 2017-09-25 at 13 05 30

**Review by rbrtsmith — APPROVED** [review]

**Pull request merged**